surveys/autonomyNeedSatisfaction/autonomyNeedSatisfaction.cy.jsx

```diff
@@ -5,11 +5,44 @@ const dummy = {
   set(response) {},
 };
 
+const dataNames = [
+  "chooseDirection",
+  "chooseContent",
+  "sharingChoice",
+  "importantDiscussion",
+  "sensitiveTopics",
+  "feelingFree",
+  "voiceOpinion",
+  "controlledPressured",
+];
+const dataNameSet = new Set(dataNames);
+
 describe("AutonomyNeedSatisfaction", () => {
   it("completes", () => {
     cy.spy(dummy, "set").as("callback");
     cy.mount(<AutonomyNeedSatisfaction onComplete={dummy.set} />);
 
+    // check that the order of the questions is randomized
```

Comment:
Here's my changes to make the test a bit more robust. I would guess line 18, and most of lines 25-43 could be pulled out and put in a custom command that would make it easy to reuse.